Documentation/dynamic-rupture.rst

```diff
@@ -153,3 +153,37 @@ Linear-Slip Weakening Friction
 
 Rate-and-State Friction
 ^^^^^^^^^^^^^^^^^^^^^^^
+
+Thermal Pressurization
+~~~~~~~~~~~~~~~~~~~~~~
+
+Following Noda & Lapusta (2010) Seissol can now account for thermal pressurization of pore fluids mimicking the effects of rapid co-seismic slip generating heat that increases temperature and pore pressure in case of low hydraulic diffusivity of the surrounding rock. Consequently, the elevated pore pressure reduces the effective normal stress, causing dynamic weakening. 
```

Comment:
Maybe you could rephrase this in shorter sentences. The part "generating heat that increases temperature" is confusing for me. Isn't generating heat and increasing temperature the same thing? 
Do we want to include a link to the Noda & Lampusta paper? How is the general rule of citations in the docu?

Reply:
In general yes it is the same. But here I state explicitly because of the two diffusion equations of temperature and pressure that are needed to be solved.
I do not know the convention.